RiotSwiftUI/Modules/Authentication/Common/Service/MatrixSDK/LoginWizard.swift

```diff
@@ -54,7 +54,7 @@ class LoginWizard {
     ///   - initialDeviceName: The initial device name.
     ///   - deviceID: The device ID, optional. If not provided or nil, the server will generate one.
     /// - Returns: An `MXSession` if the login is successful.
-    func login(login: String, password: String, initialDeviceName: String, deviceID: String? = nil) async throws -> MXSession {
+    func login(login: String, password: String, initialDeviceName: String, deviceID: String? = nil, resetOthers: Bool = false) async throws -> MXSession {
```

Comment:
I wonder if in here the parameter name should be a bit more descriptive such as `replaceOtherAccounts`. Or possibly just needs a doc comment to clarify what it does. Same thought either way for the token-based method below too.

Reply:
I've picked `removeOtherAccounts`.